internal/pkg/mevent/meta_event.go

```diff
@@ -0,0 +1,112 @@
+package mevent
```

Comment:
@Dotunj Let's move this to the `services` package. We would eventually move this to `internal/services`. We can rename it to `create_meta_event.go` as well.